src-js/services/rum.js

```diff
@@ -0,0 +1,21 @@
+import { datadogRum } from '@datadog/browser-rum';
+import { isProduction, isUnsupportedBrowser } from '../utils';
+
+export const getDefaultOptions = () => ({
+  applicationId: '2a3e19ca-297e-4a53-a2cf-082e827d13d3',
+  clientToken: 'pubdffb4b4dd07f1f43b42276901d58da91',
+  site: 'datadoghq.com',
+  service: 'circleci-docs',
+  env: isProduction() ? 'production' : 'development',
+  trackInteractions: false,
```

Comment:
Im curious, wouldn't we want `trackInteractions` interactions to be `true` to help monitor user clicks in our app in combination with setting a [custom action name](https://docs.datadoghq.com/real_user_monitoring/browser/tracking_user_actions/?tab=npm#declaring-a-name-for-click-actions) or implementing [global scrubbing](https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm) to not release sensitive data?

Reply:
Probably not. These settings are directly coming from the one chosen on a company level in: https://github.com/circleci/frontplane/blob/80b07b1f3b7db7b6d541c2a1c59f617d02a96369/packages/real-user-monitoring/src/utils.ts#L25-L26

Reply:
thats a good point, makes sense!